### ISSUE TITLE
Use directory name for external todo files

### ISSUE THREAD

**Issue by nealot** [fabiospampinato/vscode-todo-plus#140]
Currently, if you specify an external todo file (a file outside of current project) using `todo.file.include`, the file view will categorize it under the label `undefined`. It would be better if the label could be changed to the directory name of the external file.

So this:

```
"todo.file.include": [
  "/home/example/one.todo",
  "/home/example/two.todo",
  "/home/example/lists/todo.list"
]
```

Would display:

```
example
  one.todo
  two.todo
lists
  todo.list
```

**Comment by fabiospampinato** [comment]
I'll look into that 👍

**Comment by fabiospampinato** [comment]
Fixed in v4.9.2. It should work as you suggested.
